narwhals/dataframe.py

```diff
@@ -271,6 +288,8 @@ def to_pandas(self) -> Any:
         Convert this DataFrame to a pandas DataFrame.
 
         Examples:
+            Construct pandas and polars DataFranes:
```

Comment:
Can we correct the spelling on this to DataFrames?

Reply:
Done